Dockerfile

```diff
@@ -7,4 +7,10 @@ RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o bin/image-customizat
 FROM registry.ci.openshift.org/ocp/4.11:base
 COPY --from=builder /go/src/github.com/openshift/image-customization-controller/bin/image-customization-controller /
 COPY --from=builder /go/src/github.com/openshift/image-customization-controller/bin/image-customization-server /
+
+# Binarys have been renamed to machine-image-customization-*, using a symlink for
+# now to ensure backward compat, can be removed at a later stage
+RUN ln -s /image-customization-controller /machine-image-customization-controller
+RUN ln -s /image-customization-server /machine-image-customization-server
```

Comment:
I don't quite understand the comment about backwards compat.   According to the lines above, the binaries haven't been renamed? They are still being copied as image-customization-controller and image-customization-server.

Is everything now looking for the machine-* prefixed versions?

Reply:
To be clear, I think the commands are right, you're creating the missing machine- commands that https://github.com/openshift/installer/pull/5909 is failing on, I just think the comment isn't correctly explaining the motivation.